docker/table-diff-handle/tablediffview/__main__.py

```diff
@@ -0,0 +1,47 @@
+import argparse
+import os
+from tablediffview.create_diff_markdown import create_diffs
+from tablediffview.github_api_handle import (
+    get_issue_number,
+    remove_old_comments,
+    create_issue_comment,
+)
+from zipfile import ZipFile, ZIP_DEFLATED
+from io import BytesIO
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--diffDir", required=True, type=str)
+    parser.add_argument("--branchName", required=True, type=str)
+    parser.add_argument("--prNumber", required=False, type=str)
+    diff_dir = parser.parse_args().diffDir
+    branch_name = parser.parse_args().branchName
+    pr_number = parser.parse_args().prNumber
+    issue_number = ""
+    if pr_number:
+        issue_number = pr_number
+    else:
+        issue_number = get_issue_number(branch_name)
+    if not issue_number:
+        raise SystemError(
+            f"Pull Request/Issue number for branch: '{branch_name}' not found"
+        )
+    remove_old_comments(issue_number)
+    if not os.path.exists(diff_dir) or not os.listdir(diff_dir):
+        return
```

Comment:
@TruongQuangSB I would switch these two commands or at least write a comment in the github isse that the diff could not be generated due to a particular reason.

Otherwise we remove old comments and have no information anymore in the PR.

Reply:
@mariusheine  I mean here, when it give no more different, also the table referencen are alread update, then the old comment will be remove. Aside from that, when the SWT Bot fail by another Test, not Table Test, then this Image will be call anyway.

Reply:
@TruongQuangSB Ahh alright. Then we can keep it like this